doc/Makefile

```diff
@@ -43,44 +44,49 @@ help:
 	@echo "  doctest    to run all doctests embedded in the documentation (if enabled)"
 	@echo "  updaterst  to update sphinx rst to reflect code structure changes"
 
+schema.md: parse_gx_xsd.py schema_template.md ../lib/galaxy/tools/xsd/galaxy.xsd ## Build Github-flavored Markdown from Galaxy Tool XSD (expects libxml in environment)
+	python parse_gx_xsd.py schema_template.md ../lib/galaxy/tools/xsd/galaxy.xsd > $@
```

Comment:
@nsoranzo why is this called a second time (with `python`)? What does `$@` refer to - is it the make target (`schema.md`)?